pkg/auth/rbac.go

```diff
@@ -40,15 +40,16 @@ const (
 	PermissionDeleteEnvironmentApplication = "DeleteEnvironmentApplication"
 	PermissionDeployReleaseTrain           = "DeployReleaseTrain"
 	// The default permission template.
-	PermissionTemplate = "%s,%s,%s:%s,%s,allow"
+	PermissionTemplate = "p,role:%s,%s,%s:%s,%s,allow"
 )
 
 // All static rbac information that is required to check authentication of a given user.
 type RBACConfig struct {
 	// Indicates if Dex is enabled.
 	DexEnabled bool
-	// The RBAC policy. A key is a permission, for example: "Developer, CreateLock, development:development, *, allow"
-	Policy map[string]*Permission
+	// The RBAC policies. The key is a permission or group, for example: "Developer, CreateLock, development:development, *, allow"
+	// Permissions are checked in the Permissions map and Group Policies are checked in the Groups Map
```

Comment:
```suggestion
```
Not sure what the (2nd line of the) comment is really telling me, so I would either remove it, or rephrase it.
It also sounds a bit like it would fit at line 146 (type RBACPolicies)